docs/use/connect-your-wallet/metamask.mdx

```diff
@@ -4,107 +4,96 @@ sidebar_position: 3
-    - **New RPC URL:**  <Highlighter keyword="rpc_url" />
-    - **Chain ID:**  <Highlighter keyword="chain_id" />
-    - **Currency Symbol (optional):** <Highlighter keyword="ticker" />
-    - **Block Explorer URL (optional):** <Highlighter keyword="evm_explorer_url" />
-  
-- Testnet
+You can also connect Metamask to a local node, which is useful if you are
+planning on developing on Evmos locally. If you haven't already set up your own
+local node, refer to [the quickstart
+tutorial](./../../protocol/evmos-cli/single-node), or follow the instructions in
+the [GitHub repository](https://github.com/evmos/evmos/).
 
-    - **Network Name:** <Highlighter keyword="name" postText=" Testnet" />
-    - **New RPC URL:**  <Highlighter keyword="rpc_url_testnet" />
-    - **Chain ID:** <Highlighter keyword="testnet_chain_id" />
-    - **Currency Symbol (optional):**  <Highlighter keyword="testnet_ticker" />
-    - **Block Explorer URL (optional):**  <Highlighter keyword="testnet_evm_explorer_url" />
-  
-- Local Node
+Open the MetaMask extension on your browser, you may have to log in to your
+MetaMask account if you are not already. Then click the top right circle and go
+to `Settings` > `Networks` > `Add Network` and fill the form as shown below.
 
-    - **Network Name:**  <Highlighter keyword="name" postText=" Local" />
-    - **New RPC URL:**  <Highlighter keyword="rpc_url_local" />
-    - **Chain ID:**  <Highlighter keyword="rpc_url_local" />
-    - **Currency Symbol (optional):** <Highlighter keyword="testnet_ticker" />
-    - **Block Explorer URL (optional):** `n/a`
+import ProjectValue from '@site/src/components/ProjectValue';
+import Highlighter from '@site/src/components/Highlighter';
 
-## Import Account to Metamask
+![metamask networks settings](/img/metamask_network_settings.png)
 
-### Automatic Import
+Here is the list of fields that you can use to paste on Metamask:
 
-Once you have added <ProjectValue keyword="name" /> to the Metamask `Networks`, you can automatically import your accounts by:
+- **Network Name:** <Highlighter keyword="name" postText=" Local" />
+- **New RPC URL:** <Highlighter keyword="rpc_url_local" />
+- **Chain ID:** <Highlighter keyword="rpc_url_local" />
+- **Currency Symbol (optional):** <Highlighter keyword="testnet_ticker" />
+- **Block Explorer URL (optional):** `n/a`
 
-1. Go to the official EVM Chain ID Registry website: [chainlist.org](https://chainlist.org/)
-2. Search for `"Evmos"`
-   ![chainlist.org website](/img/chainlist.png)
-3. Click the `Connect Wallet` button under `Evmos Testnet`
-   ![add accounts via chainlist](/img/chainlist_metamask.png)
+## Importing Accounts
 
-### Manual Import
+If you already have an account on a Evmos network you can import it to Metamask
+using the account's private key.
 
-Close the `Settings`, go to `My Accounts` (top right circle) and select `Import Account`. You should see an image like the following one:
```

Comment:
These instructions were confusing, as I don't know where to close 'Settings'. Recommend getting rid of that part and starting with "Go to 'My Accounts'..."